flows/vision-zero/cris_import/cris_import.py

```diff
@@ -272,7 +301,14 @@ def pgloader_csvs_into_database(directory):
 @task(name="Remove trailing carriage returns from imported data")
 def remove_trailing_carriage_returns(data_loaded_token):
 
-    pg = psycopg2.connect(host=DB_HOST, user=DB_USER, password=DB_PASS, dbname=DB_NAME, sslmode="require", sslrootcert="/root/rds-combined-ca-bundle.pem")
+    pg = psycopg2.connect(
+        host=DB_HOST,
+        user=DB_USER,
+        password=DB_PASS,
+        dbname=DB_NAME,
+        sslmode=DB_SSL_REQUIREMENT,
+        sslrootcert="/root/rds-combined-ca-bundle.pem",
```

Comment:
Nice - this will make troubleshooting a bit easy if we run into any SSL issues in the future. Also, do we need to pass this cert? 

Last week, I noticed that our password manager entry called **Vision Zero Database Configuration** has a note that says
> SSL is required to connect to this service, but should not need a special certificate installed.

I forgot to ask about this then so I figured now would be a good time to check if we need to update that.